lib/services.rb

```diff
@@ -0,0 +1,12 @@
+require 'gds_api/publishing_api'
+require 'gds_api/panopticon'
+
+module Services
+  def self.publishing_api
+    @publishing_api ||= GdsApi::PublishingApi.new(Plek.new.find('publishing-api'))
+  end
+
+  def self.panopticon
```

Comment:
This should probably be called `panopticon_registerer` because it's not constructing a standard panopticon api client.

Additionally, the Registerer contains some of the information about the thing being sent to panopticon (including the `kind`, which defaults to custom_application).  For this reason, it's probably better to construct this closer to the point of registration.